counterparty-core/counterpartycore/lib/api/api_server.py

```diff
@@ -244,6 +255,8 @@ def handle_route(**kwargs):
 
 
 def run_api_server(args):
+    logger.info("Starting API Server v2.")
```

Comment:
I think we shouldn't specify the version here. It's just the default now :)

Reply:
:-) yes!

Reply:
done